roles/deploy_bmh/template/bmh.yml.j2

```diff
@@ -13,7 +13,7 @@ metadata:
     workload: {{ node_name.split('-')[0] }}
 spec:
   bmc:
-    address: {{ node_data['connection'] }}
+    address: {{ cifmw_deploy_bmh_redfish_bmc_protocol ~ '+' if 'redfish' in node_data['connection'] }}{{ node_data['connection'] }}
```

Comment:
We are using `idrac` in most of the DS hardware, this approach will blow the entire NFV job.

Reply:
I checked the D/S job and only seen ipmi addresses in the rendered bmh resource? If you could point me to a reference on Slack I would be grateful. 

You're right though, this test is too broad and would affect anyone using redfish, I'll have a think and submit a revision, thanks :)